concepticondata/conceptlists/Luangthongkum-2019-341.tsv

```diff
@@ -0,0 +1,342 @@
+Luangthongkum-2019-341-107	107	eye, face		
+Luangthongkum-2019-341-108	108	fall down	1280	FALL
+Luangthongkum-2019-341-109	109	fat (adj)		
+Luangthongkum-2019-341-110	110	feather, fur, body hair		
+Luangthongkum-2019-341-111	111	feel full (after eating)		
+Luangthongkum-2019-341-112	112	female, woman	962	WOMAN
+Luangthongkum-2019-341-113	113	female (humans), mother	1216	MOTHER
+Luangthongkum-2019-341-114	114	fire	221	FIRE
+Luangthongkum-2019-341-115	115	fish	227	FISH
+Luangthongkum-2019-341-116	116	five	493	FIVE
+Luangthongkum-2019-341-117	117	flank, rib		
+Luangthongkum-2019-341-118	118	flat	1633	FLAT
+Luangthongkum-2019-341-119	119	flea	232	FLEA
+Luangthongkum-2019-341-120	120	flesh, meat	2615	FLESH OR MEAT
+Luangthongkum-2019-341-121	121	flower	239	FLOWER
+Luangthongkum-2019-341-122	122	forget	1523	FORGET
+Luangthongkum-2019-341-123	123	forest, jungle	420	FOREST
+Luangthongkum-2019-341-124	124	four	1500	FOUR
+Luangthongkum-2019-341-125	125	fox, wolf		
+Luangthongkum-2019-341-126	126	frog	503	FROG
+Luangthongkum-2019-341-127	127	front	2194	FRONT (PART)
+Luangthongkum-2019-341-128	128	fruit	1507	FRUIT
+Luangthongkum-2019-341-129	129	full	1429	FULL
+Luangthongkum-2019-341-130	130	ginger	2500	GINGER
+Luangthongkum-2019-341-131	131	go	695	GO
+Luangthongkum-2019-341-132	132	grasshopper	607	GRASSHOPPER
+Luangthongkum-2019-341-133	133	grandchild	1617	GRANDCHILD
+Luangthongkum-2019-341-134	134	great-grandchild		
+Luangthongkum-2019-341-135	135	great-great-gandchild		
+Luangthongkum-2019-341-136	136	hair (head~)	2648	HAIR (HEAD)
+Luangthongkum-2019-341-137	137	handle	1254	HANDLE
+Luangthongkum-2019-341-138	138	hat	771	HAT
+Luangthongkum-2019-341-139	139	hawk, eagle	2683	EAGLE OR HAWK
+Luangthongkum-2019-341-140	140	have, be, exist, stay, there is		
+Luangthongkum-2019-341-141	141	head	1256	HEAD
+Luangthongkum-2019-341-142	142	heart	1223	HEART
+Luangthongkum-2019-341-143	143	heavy	1210	HEAVY
+Luangthongkum-2019-341-144	144	hemp	2041	HEMP
+Luangthongkum-2019-341-145	145	hip	1745	HIP
+Luangthongkum-2019-341-146	146	hit hard (with a fist)	353	STRIKE
+Luangthongkum-2019-341-147	147	horn	1393	HORN (ANATOMY)
+Luangthongkum-2019-341-148	148	hornet	3261	HORNET
+Luangthongkum-2019-341-149	149	horse	615	HORSE
+Luangthongkum-2019-341-150	150	hundred	1634	HUNDRED
+Luangthongkum-2019-341-151	151	husband	1200	HUSBAND
+Luangthongkum-2019-341-152	152	ill, painful, sick	2103	PAINFUL OR SICK
+Luangthongkum-2019-341-153	153	insect	620	INSECT
+Luangthongkum-2019-341-154	154	intestines	1991	INTESTINES
+Luangthongkum-2019-341-155	155	iron	621	IRON
+Luangthongkum-2019-341-156	156	itchy	149	ITCHY
+Luangthongkum-2019-341-157	157	kidney	489	KIDNEY
+Luangthongkum-2019-341-158	158	leaf	628	LEAF
+Luangthongkum-2019-341-159	159	leech (land~)	2273	LEECH
+Luangthongkum-2019-341-160	160	leech (water~)	949	WATER LEECH
+Luangthongkum-2019-341-161	161	left (side)	244	LEFT
+Luangthongkum-2019-341-162	162	leg	1297	LEG
+Luangthongkum-2019-341-163	163	lemur (flying~), flying squirrel		
```

Comment:
Perhaps 3271 FLYING SQUIRREL?

Reply:
I hesitated to map this one because of the flying lemur which appears to be a different animal. I assume, however, that they could be collapsed into one category based on their similarity

Reply:
True, they seem to be similar: "Flying lemurs resemble large flying squirrels, as they are arboreal climbers and gliders that have webbed feet with claws. The form of the head and the nocturnal habit, however, recall the lemurs, hence their name." (see [here](https://www.britannica.com/animal/flying-lemur)). 
So, I'd leave it unmapped to avoid confusion.